spec.html

```diff
@@ -596,7 +671,9 @@ <h1>%TypedArray%.prototype.slice ( _start_, _end_ )</h1>
       <emu-alg>
         1. Let _O_ be the *this* value.
         1. Perform ? ValidateTypedArray(_O_).
-        1. Let _len_ be _O_.[[ArrayLength]].
```

Comment:
Minor bookkeeping consideration: this line should be persisted but wrapped in a `<del>` tag.

Reply:
Oops, no, now I see how it's being modified in-place. Sorry for the noise!